README.md

```diff
@@ -53,6 +53,45 @@ $ docker run -it --rm --name my-running-script -v "$PWD":/usr/src/app -w
 /usr/src/app node:4 node your-daemon-or-script.js
 ```
 
+## Verbosity
+
+By default Node.js Docker Image has NPM log verbosity set to `verbose` instead
```

Comment:
"By default Node.js" -> "By default the Node.js"

`has NPM log verbosity` -> `has the npm log verbosity`